plugins/hydra_ray_aws_launcher/setup.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as fh:
+    LONG_DESC = fh.read()
+    setup(
+        name="hydra-ray-aws-launcher",
+        version="0.1.0",
+        author="Jieru Hu",
+        author_email="jieru@fb.com",
+        description="Hydra Ray AWS launcher plugin",
+        long_description=LONG_DESC,
+        long_description_content_type="text/markdown",
+        url="https://github.com/facebookresearch/hydra/",
+        packages=find_packages(exclude=["tests", "example"]),
+        classifiers=[
+            "License :: OSI Approved :: MIT License",
+            "Programming Language :: Python :: 3.6",
+            "Programming Language :: Python :: 3.7",
+            "Programming Language :: Python :: 3.8",
+            "Operating System :: OS Independent",
```

Comment:
Confirm with the Ray team what are the supported operating systems.
noxfile uses this to determine which platforms to test on.

Reply:
https://ray.readthedocs.io/en/latest/installation.html
They support MacOS and Linux, python 3.6 & 3.7, will update this.

Reply:
We can still test on 3.8, they probably do support it and forgot to update. if we run into issues we can prod them.